### PR TITLE
Added admin-users-management component

### PR DESCRIPTION
**Pull request by jamilkabir**
Added Users (admin) component. I used faker to generate some dummy data. I still have a few minor things to do but wanted to get some feedback on functionality and redundancy.

Please review when you get a chance. While reviewing have the have console open. 

Thanks!

### CODE REVIEW

File: client/components/Users.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            adminUsers: []
+        }
+        this.handleChecked = this.handleChecked.bind(this)
+        this.handleDeleteUser = this.handleDeleteUser.bind(this)
+        this.handleResetPassword = this.handleResetPassword.bind(this)
+
+    }
+
+    componentDidMount(){
+        // I'll change the code below once we confirm the users data
```
Comment (zoegreene) [inline client/components/Users.js:20]: What do you mean by confirm the users data? Thinking this should eventually end up in the seed file and this component would make a db call via Redux

File: client/components/Users.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        //Needs to be integrated with backend
+        alert("Needs to be integrated with backend")
+    }
+
+   render(){
+       let id = 0; // will change id to pk when users data is confirmed
+       return (
+           <div>
+               <h1>Users</h1>
+               <div className="adminNav">
```
Comment (zoegreene) [inline client/components/Users.js:67]: If you take a look at Adam's PR, we'll want to have the Admin navigation options show up in the top black nav bar. Likely "COLLECTION" would be the same as "PRODUCTS", we'd just insert add/edit/delete buttons on the existing AllArtwork component.

**Review by zoegreene — COMMENTED** [review]
Good start to the front end of this piece - I think we'll want to merge the auth PRs first so that you're able to make this slice functional. For instance, any random user shouldn't be able to access this section.

File: client/components/Users.js (hunk 1 above)
Comment (jamilkabir) [reply client/components/Users.js:20]: I meant, I would change it after we have the users data. I wasn't sure if someone else was working on setting up the users data but I can take care of it 🙂

File: client/components/Users.js (hunk 2 above)
Comment (jamilkabir) [reply client/components/Users.js:67]: I'll take a look into his PR

**Comment by adam-adair** [comment]
Thanks Jamil, I think all this looks good. The hover-sensitive table, v attractive. Just need to wire it up now! As Zoe points out, that will have to come after back end authorization stuff is merged in.

**Comment by jamilkabir** [comment]
> Thanks Jamil, I think all this looks good. The hover-sensitive table, v attractive. Just need to wire it up now! As Zoe points out, that will have to come after back end authorization stuff is merged in.

Sounds good!

**Review by z-rodney — CHANGES_REQUESTED** [review]
this looks good Jamil! i think it would be helpful to add to the seed file so we can start having users persisted in the database. i suggest something like name@fake.com and nameadmin@fake.com for each person on the team, and then just adding our names as the password.

**Comment by jamilkabir** [comment]
> this looks good Jamil! i think it would be helpful to add to the seed file so we can start having users persisted in the database. i suggest something like name@fake.com and nameadmin@fake.com for each person on the team, and then just adding our names as the password.

Hey Zaina. Added users to the main seed file.

**Review by z-rodney — APPROVED** [review]

**Pull request merged**